e2e-tests/tfm/cypress/support/ui/click-events/click-back-link.js

```diff
@@ -0,0 +1,11 @@
+import { clickEvents } from '../../../e2e/partials';
+
+/**
+ * clickBackLink
+ * Click the back link.
+ */
+const clickBackLink = () => {
+  clickEvents.backLink().click();
```

Comment:
After the above comment this can just import `backLink` and then be `backLink().click();`